docs/source/tt-metalium/tt_metal/environment_variables/index.rst

```diff
@@ -0,0 +1,75 @@
+.. _Environment Variables:
+
```

Comment:
Please add this doc in `docs/source/tt-metalium/index.rst` so it gets added to the side panel.